### ISSUE TITLE
Why is some integer in denominator kept in bracket?

### ISSUE THREAD

**Issue by zhaoli-IHEP** [vermaseren/form#315]
In some cases two integer denominators are not combined, so the result rational number looks strange. For example 

`symbol x;

Local expression = 22400/27/(2 + x)/(4 + x);
.sort

Local expr = expression*replace_(x,134);
.sort

print;
.end
`

This gives expr = 2800/459/(138);

**Comment by jodavies** [comment]
This is because they are still inside the internal denominator function.
If you rename this, as in
```
CFunction den;
Denominators den;
```
You can see what is happening.

You can then use something like
```
Identify den(x?number_) = 1/x;
```
to get rid of them.

**Comment by tueda** [comment]
Indeed, to me, this seems to be a "normalization" bug related to the `denom_` function. (Note that `22400/27/(2 + x)/(4 + x)` is internally `22400/27*denom_(2 + x)*denom_(4 + x)`, as Josh mentioned.)

As a workaround, you can insert
```
argument;
endargument;
```
after the replacement, which should turn remaining `denom_(integer)` to be `1/integer`.

**Comment by zhaoli-IHEP** [comment]
Thank you so much!
